app/src/views/private/components/license-banner.vue

```diff
@@ -1,65 +1,64 @@
 <script setup lang="ts">
-import { useI18n } from 'vue-i18n';
+import { initialValues, useFormFields, validate } from '@/routes/setup/form';
+import SetupForm from '@/routes/setup/form.vue';
 import { useSettingsStore } from '@/stores/settings';
-import { useServerStore } from '@/stores/server';
-import { storeToRefs } from 'pinia';
-import BannerSVG from '../../../assets/directus-bsl-banner.svg?raw';
+import { notify } from '@/utils/notify';
+import type { SetupForm as Form } from '@directus/types';
+import { useCookies } from '@vueuse/integrations/useCookies';
+import { computed, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
 
+const settingsStore = useSettingsStore();
+const cookies = useCookies(['license-banner-dismissed']);
 const { t } = useI18n();
 
-const settingsStore = useSettingsStore();
+const errors = ref<Record<string, any>[]>([]);
+const isSaveDisabled = computed(() => !form.value.email || !form.value.license);
```

Comment:
```suggestion
const errors = ref<Record<string, any>[]>([]);

const isSaveDisabled = computed(
	() => !form.value.email || !form.value.license || (form.value.project_usage === 'commercial' && !form.value.org_name),
);
```

To keep consistency of disabling button when the org field is rendered but empty 😃 

Blank line above is to appease the linter